### PR TITLE
Small SonarQube fixes 

### PR DESCRIPTION
**Pull request by Firassawan**
The fixes suggested below include removing unused imports and constants. 

Please refer to my comment in Directions.js and provide feedback before merging.

### CODE REVIEW

File: __tests__/system/Map.test.js [hunk 1]
```diff
@@ -16,10 +16,8 @@ const setUpBottom = (props={}) => {
 
 describe("Map Component", () => {
    let component;
-   let component2;
```
Comment (AlainJobs) [inline __tests__/system/Map.test.js]: Just to make sure, why did you remove this in the test though?

Comment (Firassawan) [reply __tests__/system/Map.test.js]: Basically it was not used in the test, I double checked that the test still passes after removing it.

Comment (AlainJobs) [reply __tests__/system/Map.test.js]: Makes sense!
